media/transcoder/src/test/java/com/example/transcoder/CreateJobFromTemplateTest.java

```diff
@@ -114,7 +114,11 @@ public void beforeTest() throws IOException {
     Path path = Paths.get(TEST_FILE_PATH);
     storage.create(blobInfo, Files.readAllBytes(path));
 
-    DeleteJobTemplate.deleteJobTemplate(PROJECT_ID, LOCATION, TEMPLATE_ID);
+    try {
+      DeleteJobTemplate.deleteJobTemplate(PROJECT_ID, LOCATION, TEMPLATE_ID);
+    } catch (Exception e) {
+      // Don't worry if the template doesn't already exist.
```

Comment:
I don't know how I feel about this. We should be deterministic in our tests.

Reply:
The try block is there only to delete the job template if it exists. This is a very rare case since the test template uses a random UUID each time. I updated the catch block to only catch the common case where the template is not found. All other errors should fail the test.